syntax = "proto3";

package tesla.proto.energy_device.v1;

option go_package = "github.com/teslamotors/energy_device/pkg/protocol/protobuf/energydevice/v1";
option java_package = "com.tesla.generated.energy_device.v1";
option java_outer_classname = "TEGAPIScheduleManualBackupEventRequest";

import "tesla_api/protobuf/energy_device/v1/control_event_scheduling_info.proto";

message TEGAPIScheduleManualBackupEventRequest {
  ControlEventSchedulingInfo scheduling_info = 1;
}